project/Dependencies.scala

```diff
@@ -186,4 +189,13 @@ object Dependencies {
     Seq(specs2core, typeSafeConfig)
   }
 
+  def databaseDependencies: Seq[ModuleID] = {
+    lazy val scalaTest  = "org.scalatest"   %% "scalatest"  % Versions.scalatest  % "test"
+    lazy val balta = "za.co.absa" %% "balta" % Versions.balta
```

Comment:
```suggestion
    lazy val balta = "za.co.absa" %% "balta" % Versions.balta % Test
```

Isn't `balta` also used for tests only?